components/support/nimbus-fml/src/backends/swift/templates/FeatureManifestTemplate.swift

```diff
@@ -104,4 +121,4 @@ public class {{ features_object }} {
 {{- code }}
 {%- endfor %}
 
-{% import "macros.kt" as kt %}
\ No newline at end of file
+{% import "macros.swift" as s %}
```

Comment:
Well this is weird. It seems like no macros were being used in this file. It should read:

```suggestion
{% import "macros.swift" as swift %}
```

or deleted.